test/common/config.js

```diff
@@ -49,7 +49,7 @@ export default function () {
         `--optimize.bundleDir=${OPTIMIZE_BUNDLE_DIR}`,
       ],
       serverArgs: [
-        '--env=development',
+        '--env.name=development',
```

Comment:
Would you mind explaining why this change was needed?

Reply:
see https://github.com/elastic/kibana/pull/18967/files#r193932359